watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1595_photothermal_membrane_candoP/amo_1595.py

```diff
@@ -264,10 +271,25 @@ def add_costing(m):
             m.fs.costing.N2O_byproduct_mass * m.fs.costing.nitrous_oxide_cost,
             to_units=m.fs.costing.base_currency / m.fs.costing.base_period,
         ),
-        doc="Dollar value of byproduct nitrous oxide generated per year",
+        doc="Value of byproduct nitrous oxide generated per year",
     )
 
-    m.fs.costing.LCOW_bcp = Expression(
+    m.fs.costing.LCOW = Expression(
+        expr=(
+            m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
+            + m.fs.costing.total_operating_cost
```

Comment:
Perhaps the costing package should provide an expression for this?

Reply:
What is this -- `total_annualized_cost`? Should probably be in both packages. Let's open an issue instead.

Reply:
Created issue #710.